lib/bulk_processor.rb

```diff
@@ -1,3 +1,6 @@
+require 'bulk_processor/back_end'
+require 'bulk_processor/back_end/active_job'
+require 'bulk_processor/back_end/dynosaur'
```

Comment:
I could override `Config#back_end=` and only require the one specified. Since these are pretty small classes, I'll defer adding complexity to handle that for now.